examples/jsm/loaders/LUTImageLoader.js

```diff
@@ -0,0 +1,160 @@
+// http://download.autodesk.com/us/systemdocs/help/2011/lustre/index.html?url=./files/WSc4e151a45a3b785a24c3d9a411df9298473-7ffd.htm,topicNumber=d0e9492
+// https://community.foundry.com/discuss/topic/103636/format-spec-for-3dl?mode=Post&postID=895258
```

Comment:
It looks like these comments are left over from referencing the LUT3dlLoader?

Reply:
Removed